app/models/pending_induction_submission_batch.rb

```diff
@@ -28,7 +28,8 @@ def self.new_action_for(appropriate_body:, **)
 
   # Associations
   belongs_to :appropriate_body
-  has_many :pending_induction_submissions
+  has_many :pending_induction_submissions, dependent: :destroy
+  has_many :events, dependent: :destroy
```

Comment:
Does it **ever** make sense to destroy Events? 🤔

Philosophical!

I wonder whether we could use something like `dependent: :nullify` so that we can safely delete the `PendingInductionSubmissionBatch` without losing potentially useful information in the form of the `Event` record.

What do you think?

Reply:
It'll be a single event most of the time, for example `AB started a bulk claim`. Good idea to err on the side of caution though, will tweak.

As for the principal of deleting events linked to teachers and inductions that the upcoming work to purge will remove, would you destroy or nullify?